### DISCUSSION TITLE
Change sign-in screen logo and appearance ?

### DISCUSSION THREAD

**Discussion by slfhstr** [Budibase/budibase#1639] [Ideas]
I see how to change the organisation logo.
But users don't see that until after they sign in from the budibase login screen.
<img width="564" alt="Screenshot 2021-06-06 at 11 52 26" src="https://user-images.githubusercontent.com/28682881/120921856-c4ee4400-c6bd-11eb-94f7-9b9543a9387d.png">

In today's internet environment, this can be a concern.
Users are (thankfully) becoming more alive to phishing and hijacked logins.
So sending them to a URL which is org-specific and org-branded (e.g. my own welcome screen), then sending them to a login screen which looks like nothing like the org, is a bit of concern.

Can the organisation logo be shown in place of the budibase space helmet ?
And allow a welcome paragraph below it ?
Change background colour ?

Can't see anything in docs (but I am not fully trusting docs yet following switch from old environment).

I have no problem giving Budibase credit, e.g. powered by Budibase, or some other credit.  
No convinced that relying on training, education or familiarity are answers to the problem.

**Comment by slfhstr** [comment]
I see from answer to first question in #1622 that it is not possible to change login screen.
If it is just it is not high enough on the 'todo' list, then understandable because you have a lot on with the new platform. 
If this is a strategic decision, I think it is a big mistake with limited justifications.

**Reply by shogunpurple** [reply]
@timconsidine - thanks for your input. It is on the todo list, and it will eventually be possible to change the login screen in several ways with the new platform. 

We will make sure that the community knows when these features are available in the builder.

**Reply by slfhstr** [reply]
Thanks @shogunpurple 
Great to hear.

Look forward to the announcement, but I think I may have to stop using Budibase.
I can't send users from a fully branded page to a 3rd party screen, back to partially branded page (but potentially not needed intermediary screen if only 1 app) then to a fully branded app.
And I can't wait 4-8-12-?? weeks to launch a system.

Huge shame when you already have the code to link to a org-specified logo and org-specified organisation name (you're doing it already on the app listing screen).

No problem leaving it as default background colour and not having any org-specific text.
No problem sticking a 'Powered by Budibase' with logo and site link (which cannot be removed)
No problem tightly controlling the user model.

**Comment by slfhstr** [comment]
I know there might be a lot of "10 min changes" but any app developer, whether intranet or extranet use case, is likely to want to see their deployment branded.

<img width="569" alt="Screenshot 2021-06-06 at 22 33 31" src="https://user-images.githubusercontent.com/28682881/120940919-a9178c00-c717-11eb-89b9-92624456be9b.png">

**Comment by mjashanks** [comment]
Also related to this - ability to customise the favicon : https://github.com/Budibase/budibase/discussions/1578
